### PR TITLE
Added new blog posts

### PR DESCRIPTION
**Pull request by blrhc**
Fixes some minor errors in the repo and adds four new draft blog posts (three of which are ready for review).

### CODE REVIEW

File: blog/announcing-new-branding/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+-   The logo was inconsistent (sometimes it was our old elephant logo, sometimes a wordmark, sometimes an emblem and occasionally combinations of all three)
+-   Logo usage guidelines were unclear in terms of positioning, backgrounds, etc.
+-   Key brand colors and typefaces were undefined
+-   There were no rules for photo style or illustration style in our communications (e.g. blog posts)
+-   Icons and other assets lacked a consistent style
+-   Assets for certain purposes simply did not exist
+-   We had no established brand design guidelines or a style guide
+
+You can see [more of our discussion](https://github.com/joystream/design/issues?q=is%3Aissue+is%3Aclosed) about these issues in our design repo on GitHub.
```
Comment (bwhm) [inline blog/announcing-new-branding/README.md:42]: Consider [including open issues](https://github.com/joystream/design/issues?q=is%3Aissue) as well? 

Alternatively, you can add labels to the relevant ones (we should focus more on this anyway), and link to those.

File: blog/announcing-new-branding/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+EMBEDDED PDF HERE
+
+
+
+
+```
+
+If you're using an older web browser, the PDF might not display correctly. If this is the case, please click [here](http://151.236.222.69/assets/brandguide.pdf) to download the guide and view it locally on your device.
```
Comment (bwhm) [inline blog/announcing-new-branding/README.md:79]: Maybe add a comment here so you remember to change URL.

File: blog/announcing-new-branding/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+* * * * *
+
+#### Disclaimer
+
+All forward looking statements, estimates and commitments found in this blog post should be understood to be highly uncertain, not binding and for which no guarantees of accuracy or reliability can be provided. To the fullest extent permitted by law, in no event shall Joystream, Jsgenesis or our affiliates, or any of our directors, employees, contractors,  service providers or agents have any liability whatsoever to any person  for any direct or indirect loss, liability, cost, claim, expense or  damage of any kind, whether in contract or in tort, including negligence, or otherwise, arising out of or related to the use of all or  part of this post, or any links to third party websites.
+
+#### Preview
+
+http://151.236.222.69/p/dca19604-608b-4d30-bca5-34542a545eed/
```
Comment (bwhm) [inline blog/announcing-new-branding/README.md:89]: I see now that the preview looks completely differently when using the link above as compared https://blog.joystream.org/p/dca19604-608b-4d30-bca5-34542a545eed/

Just FYI...

File: blog/announcing-new-branding/README.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+#### Social media card cover
+
+<p align="center"><img src="announcing-new-branding.png"></p>
```
Comment (bwhm) [inline blog/announcing-new-branding/README.md:93]: Is this @bedeho approved?
Looks a little bland for a twitter cover, and the `J` will be very small on a cellphone?

File: README.md [hunk 5]
```diff
@@ -44,16 +44,26 @@ A copy of all posts from our [ghost](https://github.com/TryGhost/Ghost) [blog](h
...
-| [Sparta - Install the UI on Windows/Mac](/blog/drafts-unpublished/sparta-ui)         | /p/8e8f08a6-08d5-4a64-9f08-08aedd0d5719/ | Unpublished       | Release - Sparta      |
-| [Supply Signature Temporarily](/blog/drafts-unpublished/supply-signature-temporarily)| /p/dbbfe66d-4af3-42f3-93fb-9cde97721988/ | Unpublished       | Release - Sparta      |
+| [The Rome Incentive Structure](/blog/rome-incentives)         | /p/c8c9f496-5808-4cf4-9b96-10dcf192877c/ | Unpublished       | Release - Rome      |
+| [Upcoming Joystream Testnets](/blog/upcoming-joystream-testnets)| /p/1f74389f-06fd-43e5-a331-6f66832dfd43/ | Unpublished       | General      |
+| [Announcing New Brand Identity](/blog/announcing-new-branding)         | /p/dca19604-608b-4d30-bca5-34542a545eed/ | Unpublished       | General      |
+| [Rome Released](/blog/rome-released)| /p/1a79a5af-786a-4978-8f8f-ad15dcf56bd5/ | Unfinished Draft       | Release - Rome      |
+
+
+
+#### Aborted Planned Posts
```
Comment (bwhm) [inline README.md:62]: Make sure to either download the atom markdown extension I'm using, or create the ToC manually. (This header is missing)

**Review by bwhm — CHANGES_REQUESTED** [review]
The branding post itself looks good, with only minor comments!

Not to be difficult, but I think it would be better to have three different PRs. We can wait with the other two until we're closer to release anyway...

(I am not reviewing the other two blog posts yet)

File: blog/announcing-new-branding/README.md (hunk 3 above)
Comment (blrhc) [reply blog/announcing-new-branding/README.md:89]: Yes, that's to be expected as the new theme hasn't been installed on blog.joystream.org yet.

File: blog/announcing-new-branding/README.md (hunk 4 above)
Comment (blrhc) [reply blog/announcing-new-branding/README.md:93]: Will ask!

File: blog/announcing-new-branding/README.md (hunk 2 above)
Comment (blrhc) [reply blog/announcing-new-branding/README.md:79]: Good suggestion :)

File: blog/announcing-new-branding/README.md (hunk 1 above)
Comment (blrhc) [reply blog/announcing-new-branding/README.md:42]: Will do!

File: README.md (hunk 5 above)
Comment (blrhc) [reply README.md:62]: Ok, apologies for overlooking that.

**Pull request closed without merging**